spec/index_spec.rb

```diff
@@ -372,4 +372,51 @@
       )
     end
   end
+
+  describe Daru::CategoricalIndex do
+    before do
+      @idx = Daru::CategoricalIndex.new(
```

Comment:
@lokeshh I found [this resource](http://betterspecs.org/#subject) very helpful for questions about rspec during my GSoC last summer. From there it seems that unlike `let`, `subject` can be unnamed (which I actually find is more confusing, but I'm no expert).